exercises/say/Say.example

```diff
@@ -0,0 +1,131 @@
+module Say exposing (say, SayError(..))
+
+import Array
+import Dict
+
+
+type SayError
+    = Negative
+    | TooLarge
+
+
+say : Int -> Result SayError String
+say number =
+    if number < 0 then
+        Err Negative
+    else if number < 20 then
+        Ok (saySmall number)
+    else if number < 100 then
+        if number % 10 == 0 then
+            Ok (saySmall number)
+        else
+            Ok (sayTens (number // 10 * 10) ++ "-" ++ (saySmall (number % 10)))
+    else if number < k then
```

Comment:
full words would be good too. it took me a second to figure out that they were abbreviations for the SI prefixes